src/zhinst/toolkit/helpers/sequences.py

```diff
@@ -379,67 +379,88 @@ class SimpleSequence(Sequence):
 
     buffer_lengths = attr.ib(default=[800], validator=attr.validators.instance_of(list))
     delay_times = attr.ib(default=[0])
+    wait_samples_updated = attr.ib(default=[0])
+    dead_samples_updated = attr.ib(default=[0])
 
     def write_sequence(self):
-        self.sequence = SequenceCommand.header_comment(sequence_type="Simple")
+        # Call the method from parent class `Sequence` and then overwrite it
+        super().write_sequence()
+        # Update the sequence type information in the header
+        self.sequence = self.sequence.replace("None", SequenceType.SIMPLE.value, 1)
```

Comment:
Replace with help function and search for sequence type only.

Reply:
The helper function and a related test is added.
`SimpleSequence` class is also updated to use this new helper function.